### DISCUSSION TITLE
Sidekick Client Error

### DISCUSSION THREAD

**Discussion by utkonos** [Vector35/binaryninja-api#5178] [Q&A] [answered]
I am opening this just for someone who encounters this, not to file a bug.

If you run into this error:
```none
[Sidekick] Sidekick encountered a client request error.
[Sidekick]
[Sidekick] Exception=422 Client Error: Unprocessable Entity for url: https://api.sidekick.binary.ninja/api/v2/notebooks/chat
```

Make sure that all three of these Python Binja settings are set to the defaults:
1. Python Path Override
2. Python Interpreter
3. Python Virtual Environment Site-Packages

When using Homebrew Python 3.11.8 on macOS, the error appears. If the bundled Python is used, all Sidekick features work without error.

**Comment by kristopax** [comment, marked as answer]
First, ensure that you are using the latest version of Sidekick. You can go to the Plugin Manager within Binary Ninja and search for the Sidekick plugin to see if there is an update available.

Secondly, if you are using a version of Python other than the current supported (which is the version that is bundled with Binary Ninja or Python 3.10 if you're on Linux), then you need to ensure that the version of pydantic installed in your Python Site-Packages is:

`pydantic>=1.10,<2`

If you still encounter the error after trying these steps, then file an issue, and we will investigate.
